README.md

```diff
@@ -10,3 +10,9 @@ Please download, unzip, and copy the corresponding asset folder for your platfor
 * Android - `/android/bin/release/`
 * Win32 - The win32 binaries are not included in the zip assets. They are available on download on [nuget](https://www.nuget.org/packages/Microsoft.PlayFab.PlayFabParty.Cpp.Windows).
 * Linux - `/x64/Release/libparty.so`
+
+## Troubleshooting
+
+### WSL Compatibility
+
+While WSL can be used as a build platform to create Party binaries, it is not supported as a runtime platform to use Party SDK and its features. A machine with a Linux OS is required to take advantage of all Party features.
```

Comment:
```suggestion
While WSL can be used to build projects which depend on PlayFab Party binaries, it is not supported as a runtime platform to use the Party SDK. A machine with a Linux OS is required to take advantage of all Party features.
```
since we don't support anyone actually creating their own Party binaries

Reply:
I'll just remove that first part of the sentence entirely.